js/src/ability/defineAbility.js

```diff
@@ -0,0 +1,15 @@
+import { defineAbility } from '@casl/ability';
+
+export const defineUserAbility = (role) => {
+  return defineAbility((can) => {
+    role = role?.toLowerCase().trim();
+
+    if (role === 'admin') {
```

Comment:
Same here, have a constant for roles and import

Reply:
created a const file